projects/README.md

```diff
@@ -12,4 +12,6 @@
 |[PostgreSQL](http://www.postgresql.org/ 'PostgreSQL')|[open](http://www.postgresql.org/about/licence 'PostgreSQL license')|PostgreSQL redistributable binaries|9.4.9|[repo](https://github.com/postgres/postgres 'Mirror of the official PostgreSQL GIT repository on github')|none|
 |[Qt](http://qt.io/ 'Qt - Home')|[LGPL](http://www.qt.io/qt-licensing-terms/ 'LGPL')|One Qt Code: Create Powerful Applications & Devices|5.8.0|[repo](http://code.qt.io/cgit/qt/qt5.git 'Qt5 main repo')|[diff](https://github.com/distributePro/ovsrpro/blob/master/patches/qt5.patch 'patch/diff')|
 |[Qwt](http://http://qwt.sourceforge.net/ 'Qwt - Qt Widgets for Technical Applications')|[LGPL](http://qwt.sourceforge.net/qwtlicense.html 'LGPL with exceptions')|The Qwt library contains GUI Components and utility classes which are primarily useful for programs with a technical background.|6.1.2|none|[diff](https://github.com/distributePro/ovsrpro/blob/master/patches/qwtconfig.pri.patch 'patch/diff')|
+|[ZeroMQ](https://zeromq.org/ 'ZeroMQ - Home")|[LGPL](https://github.com/zeromq/libzmq/blob/master/README.md/ 'LGPL v3')|ZeroMQ is a lightweight messaging kernel that extends standard socket interfaces with more powerful features.|4.2.1|[repo](https://github.com/zeromq/libzmq.git)|none|
```

Comment:
Below it looks like version 4.2.2 is the one that is actually being built - this version should be updated to match